### ISSUE TITLE
Investment parameters

### ISSUE THREAD

**Issue by david-gauquelin** [vojtechsimetka/planuj.finance#227]
**Entry fee**
Set as a percentage of the total amount to be deposited.
*→ What happens when this changes? For example when a deposit is created, edited or deleted?*
Entry fee can be paid in 3 manners:
- Ongoing (you pay the specified % as a % of each deposit)
- 40/60 (you pay 40% fee on each deposit and 60% goes to your investment, until you paid the total fee)
- Upfront (100% of your deposits go to pay the fee until you paid the full fee. Only after that your deposit will actually go to your investment)
*→ The only thing that doesn't really make sense to me with 40/60 and Upfront is that you pay a fee on a total of money you "plan to" deposit, not sure what happens if you deposit less or more?*

**Exit fee**
Can be set as:
- Percentage (you pay the specified % as a % of each withdrawal)
- Fixed fee (you pay the specified fixed amount with each withdrawal)

**Success fee**
*→ Paid monthly on the amount of interest earnt? Not sure...*

**Management fee**
Can be set as:
- Percentage
*→ % of portfolio value?*
*→ paid each month?*
- Fixed fee (fixed amount)
*→ paid each month?*

**Comment by vojtechsimetka** [comment]
> *→ What happens when this changes? For example when a deposit is created, edited or deleted?*

This gets recalculated. Thats not a problem.

>   _→ The only thing that doesn't really make sense to me with 40/60 and Upfront is that you pay a fee on a total of money you "plan to" deposit, not sure what happens if you deposit less or more?_

You generally pay the whole fee even if you deposit less. If you deposit more there may be some additional fees but that is definitely not part of the MVP.

> *→ Paid monthly on the amount of interest earnt? Not sure...*

I'll investigate how exactly it is calculated (because there is a difference if it is charged monthly, quaterly or yearly). But for sure it is percentage.

> **Management fee**
> Can be set as:
> - Percentage
> *→ % of portfolio value?*
> *→ paid each month?*
> - Fixed fee (fixed amount)
> *→ paid each month?*

Yes. At least for MVP for sure.

**Comment by agazso** [comment]
Closing as completed
